spec/index.bs

```diff
@@ -7213,9 +7218,21 @@ GPUQuerySet includes GPUObjectBase;
         The {{GPUQuerySetDescriptor}} describing this query set.
 
         All optional fields of {{GPUTextureViewDescriptor}} are defined.
+
+    : <dfn>\[[state]]</dfn> of type [=query set state=].
+    ::
+        The current state of the {{GPUQuerySet}}.
 </dl>
 
-### Creation ### {#queryset-creation}
+Each {{GPUQuerySet}} has a current <dfn dfn>query set state</dfn> on the [=Content timeline=]
+which is one of the following:
+
+ - "<dfn dfn for="query set state">available</dfn>" where the {{GPUQuerySet}} is
+     available for GPU operations on its content.
+ - "<dfn dfn for="query set state">destroyed</dfn>" where the {{GPUQuerySet}} is
+     no longer available for any operations except {{GPUQuerySet/destroy}}.
```

Comment:
did we discuss this anywhere? seems surprising to be

Reply:
This was not discussed in GPUQuerySet, but should had been discussed in GPUBuffer, because it's already like this.
> "destroyed" where the GPUBuffer is no longer available for any operations except destroy.